pkg/apply/event/event.go

```diff
@@ -129,9 +132,11 @@ const (
 )
 
 type PruneEvent struct {
-	Type      PruneEventType
-	Operation PruneEventOperation
-	Object    runtime.Object
+	Type       PruneEventType
+	Operation  PruneEventOperation
+	Object     runtime.Object
```

Comment:
I think we can just make this `*unstructured.Unstructured`. I think we are using a client that will always return unstructured objects, so we can probably make it easier for clients by using a concrete type here.

Reply:
That sounds good to me.

Reply:
Changed it to `*unstructured.Unstructured`.